docs/08-cicd.md

```diff
@@ -1,80 +1,190 @@
+### Android
 
-// PEM is base64 encoded
-// Disable "Include Administrators" in Protected Branches (GitHub App is Admin)
+For an overview of the Fastlane lanes refer to the [auto-generated README](../android/fastlane/README.md#available-actions). These lanes are responsible for setting up the signing keys and building the Android app.
 
+### iOS
+
+For an overview of the Fastlane lanes refer to the [auto-generated README](../ios/fastlane/README.md#available-actions). These lanes are responsible for setting up the certificates and building the iOS app.
+
+# Setup Certificates for iOS on Mac
+
+The setup of certificates is demonstrated in section [Manual builds](06-manual-builds.md#manually-building-for-ios).
+
+## What Types of Certificates Exist?
+
+The certificate system of Apple seems very complex at first. Therefore, we will briefly explain the types of certificates and what the purpose is.
+
+|Certificate Type|Usable for Development|Usable for Testing|Usable for Production|
+|---|---|---|---|
+|Development|:heavy_check_mark:|:x:|:x:|
+|Ad-hoc|:x:|:heavy_check_mark:|:x:|
+|Enterprise|:x:|:heavy_check_mark:|:heavy_check_mark:|
+|App Store|:x:|:x:|:heavy_check_mark:|
+
+Also, there are various ways to install an iOS app:
+
+* Via a URL which is opened in Safari
+* Via App Store
+* Via XCode
+
+Development certificates are for developers who want to install the app on their devices. You have to select the devices in the Provisioning Profile at [developer.apple.com](https://developer.apple.com/account/resources/profiles/list) you want to install the app on. This means the Device ID (UDID) has to be whitelisted.
+
+Ad-hoc certificates are for public tests with a small amount of users. The difference is that you can provide the testers a URL which they can open on their device to install it. The Device ID (UDID) still has to be whitelisted.
+
+Enterprise certificates allow you to deliver you app within your company or have a bigger public test. It is not possible for small organisation to acquire such a certificate.
+
+App Store certificates allows the upload of the app into the App Store. It is not possible to install a app store signed app directly on your device. It has to be uploaded to App Store Connect.
+
+# Setup Java Keystore for Android
+
+The setup of the JKS is demonstrated in section [Manual builds](06-manual-builds.md#manually-building-for-android).
+
+# Determine the Next Version
+
+The next version of the app must be determined programmatically. The tool `yarn bump-version` can be used. If you execute it without parameters you will see the usage.
+
+As the version number is based on the release date, the first versions of 2020 are `2020.1.0`, `2020.1.1`, `2020.1.2`...
+
+The first release february will have the version number `2020.2.0`. 
```

Comment:
```suggestion
The following schema is used for versioning: `YYYY.mm.<release_number>`, where `<release_number>` is a counter for the number of releases in the corresponding month. E.g. the first release in January 2020 uses `2020.01.1`, the second `2020.01.2`.
```

Reply:
We start at 0. I modified the docu here.